Makefile

```diff
@@ -87,8 +87,15 @@ check-parallel:
 		xargs -0 grep -F -n "t.Parallel()" || \
 		! echo "Error: all the go tests should be run in serial."
 
-clean: failpoint-disable
+clean: failpoint-disable clean_tmp_test_data
 	$(GO) clean -i ./...
+	
+clean_tmp_test_data:
```

Comment:
It has been fixed by #49132. we can fix it in the tests. so can we remove it?

Reply:
@hawkingrei  Thank you for pointing out. I agree that we should fix the tests. I should have checked the file histories. I'd like to discuss what should be done in this PR.

I treated 3 type of folder here. I understood `3. extract` was completely fixed on the PR you mentioned. I removed  change for `3. extract`. 
6bd091f

1. test_coverage
2. replayer
3. extract

For `1. test_coverage`, there were no changes in the PR. Is it okay to leave my change?

Regarding `2. replayer`, I think some cases were missed in the fix. I still can see some untracked files.  

```
$ git status
On branch master
Your branch is up to date with 'origin/master'.

Untracked files:
  (use "git add <file>..." to include in what will be committed)
        pkg/domain/replayer/

nothing added to commit but untracked files present (use "git add" to track)

# my master branch contains commits for the fix.
$ git branch --contains a6c6042|grep master 
* master
```

I tried to fix the reamining issues for `replayer` folder. Could you review this additional change? [a85bb6e](https://github.com/pingcap/tidb/pull/49314/commits/a85bb6e7b6c95a808e69dab0a07ee829589a267d) 
`planReplayerTaskDumpWorker.HandleTask` method also call `replayer.GeneratePlanReplayerFile`. We have to care test cases which call `planReplayerTaskDumpWorker.HandleTask`.